docs/howtoguides/fix_how_to_know_what_the_fix_command_would_change.rst

```diff
@@ -1,7 +1,7 @@
 .. _pro-fix-dry-run:
 
-How to know what the ``fix`` command would change?
-**************************************************
+How to preview the result of pro fix
```

Comment:
```suggestion
How to preview the result of the ``pro fix`` command
```

For consistency. I think monospace is used in the pro fix explanation, for example.

Reply:
It is, but monospace isn't rendered in headers very well. If the inconsistency looks weird to you, then I'm sure it will to our readers as well, so I'll change it.